### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [Laura888/f1-3-c2p1-colmar-academy#4]
https://github.com/Laura888/f1-3-c2p1-colmar-academy/blob/0d6c7de6f1a3cc36adf5120a944c0f2a524cb818/Colmar%20Academy/index.html#L62

All images should have alt text, not only does this help the visually impaired but the site will not pass HTML validation without it.
